### PR TITLE
Add Healthkit schemas

### PR DESCRIPTION
**Pull request by mpgxvii**
- Add generic HealthKit schemas and specifications for different data types

### CODE REVIEW

File: commons/active/healthkit/healthkit_typed_data.avsc [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      "type": ["null", "double"],
+      "doc": "Value of the double data retrieved.",
+      "default": null
+    },
+    {
+      "name": "stringValue",
+      "type": ["null", "string"],
+      "doc": "Value of the string data retrieved.",
+      "default": null
+    }
```
Comment (yatharthranjan) [inline commons/active/healthkit/healthkit_typed_data.avsc:65]: are there any other types as well like Boolean?

File: specifications/active/aRMT-healthkit-1.0.0.yml [hunk 2]
```diff
@@ -0,0 +1,82 @@
+name: aRMT
+vendor: RADAR
+model: aRMT-App
+version: 1.0.0
+assessment_type: QUESTIONNAIRE
+doc: aRMT Healthkit data definition.
```
Comment (yatharthranjan) [inline specifications/active/aRMT-healthkit-1.0.0.yml:6]: Can you maybe elaborate more here, that we are using a generic schema for all data types and how it has to be interpreted?

File: specifications/active/aRMT-healthkit-1.0.0.yml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+data:
+  - doc: A quantity sample type that measures the number of steps the user has taken, unit is count.
+    topic: active_healthkit_steps
```
Comment (yatharthranjan) [inline specifications/active/aRMT-healthkit-1.0.0.yml:9]: should we have `apple` in the topic name (likely replace `active`)?

File: commons/active/healthkit/healthkit_typed_data.avsc (hunk 1 above)
Comment (mpgxvii) [reply commons/active/healthkit/healthkit_typed_data.avsc:65]: I think mostly the values are numerical. I haven't seen a boolean value in the docs.

File: specifications/active/aRMT-healthkit-1.0.0.yml (hunk 3 above)
Comment (mpgxvii) [reply specifications/active/aRMT-healthkit-1.0.0.yml:9]: Should we make it `active_apple_healthkit_steps`? So we know it's from the active app?

**Review by yatharthranjan — APPROVED** [review]
LGTM.

**Pull request merged**